src/server/lib/devHttpsAgent.ts

```diff
@@ -0,0 +1,25 @@
+import { Agent } from 'https';
```

Comment:
it should be possible to get the self signed certificate confirmed with local node. This currently works with the Scala apps.

Reply:
Ah interesting! I've noticed that the certs for `gulocal.com` work great on my macbook, however on windows/linux (ubuntu) the certs aren't automatically whitelisted using `dev-nginx`, I'll play around further.